### PR TITLE
Student name in emails are not HTML-sanitized #5698

### LINKED ISSUE #5698: Student name in emails are not HTML-sanitized
As per title. Caught when trying to generate some emails.

Thanks @bamboo3250 for providing the student name `student1 In Course1</td></div>'"` :smirk:

### PR DESCRIPTION
**Pull request by leeyimin**
Fixes #5698

### CODE REVIEW

**Comment by wkurniawan07** [comment]
@leeyimin is this ready for review?

**Comment by leeyimin** [comment]
@wkurniawan07 Sorry haven't gotten round to finish writing the tests. Should be done by tmr!

**Comment by leeyimin** [comment]
Closing and reopening to restart test.

**Comment by leeyimin** [comment]
@wkurniawan07 Ready for review.

**Comment by wkurniawan07** [comment]
@ablyx time for your first peer review!

File: src/main/java/teammates/common/util/Templates.java [hunk 1]
```diff
@@ -10,17 +10,18 @@ private Templates() {
...
+     * with the given value strings.
      * @param template The template html to be populated
-     * @param values Array of a variable, even number of key-value pairs:
+     * @param keyValuePairs Array of a variable, even number of key-value pairs:
      *                   { "key1", "val1", "key2", "val2", ... }
      * @return The populated template
      */
-    public static String populateTemplate(String template, String... values) {
-        Assumption.assertTrue("The number of values passed in must be even", values.length % 2 == 0);
+    public static String populateTemplate(String template, String... keyValuePairs) {
```
Comment (ablyx) [inline src/main/java/teammates/common/util/Templates.java]: would a map be better?

Comment (leeyimin) [inline src/main/java/teammates/common/util/Templates.java]: Hmm i think you are right but such a change would involve too many other files. I will open a new issue for refactoring it.

File: src/test/java/teammates/test/cases/util/SanitizerTest.java [hunk 2]
```diff
@@ -72,24 +72,6 @@ public void testSanitizeForHtml() {
         sanitizeHtml_receivesCodeInjection_returnsSanitized();
         sanitizeHtml_receivesSanitized_returnsUnchanged();
     }
-
```
Comment (wkurniawan07) [inline src/test/java/teammates/test/cases/util/SanitizerTest.java]: The way I understands it, it looks like this file should not be changed at all?

**Comment by leeyimin** [comment]
@wkurniawan07 Changes to that file reverted. Ready for review.

**Pull request merged**